jde/modules/ROOT/pages/demo_poll_edi_events.adoc

```diff
@@ -169,183 +181,262 @@ If you are having trouble resolving all dependencies,
+. In the Mule Palette, either select *Core*, scroll down to *Transformers* or type *Payload* in the search bar.
 
 image:demo_poll_edi_events/image19_demo_poll_edi_events.png[image,width=277,height=209]
 
-Drag and drop the _Set Payload_ to your canvas.
+[start=2]
+
+. Drag and drop the *Set Payload* to your canvas.
 
 image:demo_poll_edi_events/image20_demo_poll_edi_events.png[image,width=365,height=192]
 
-Select the _Set Payload_ component, and review the properties.
+[start=3]
+
+. Select the *Set Payload* component, and review the properties.
 
 image:demo_poll_edi_events/image21_demo_poll_edi_events.png[image,width=601,height=157]
+[start=4]
 
-Change the payload to reflect the desired output, and save the project
+. Change the *Payload* to reflect the desired output, and save your project.
 
 image:demo_poll_edi_events/image22_demo_poll_edi_events.png[image,width=601,height=216]
 
-On the _MIME Type_ tab, select _application/xml_
+[start=5]
+
+. On the *MIME Type* tab, select *application/xml*.
 
 image:demo_poll_edi_events/image23_demo_poll_edi_events.png[image,width=399,height=216]
 
-Next we need to check that Transactions were polled, and exist. From the Mule Palette, Select and drag the _Choice_ component.
+Check that the Transactions were polled. 
+
+. From the Mule Palette, select and drag the *Choice* component.
 
 image:demo_poll_edi_events/image24_demo_poll_edi_events.png[image,width=344,height=255]
 
 image:demo_poll_edi_events/image25_demo_poll_edi_events.png[image,width=600,height=259]
 
-Select the _When_ statement, and review the properties. Enter the below expression to check that transactions exist.
+[start=2]
+
+. Select the *When* statement, and review the properties.
+. Enter the following expression to check that the transactions exist.
 
 image:demo_poll_edi_events/image26_demo_poll_edi_events.png[image,width=545,height=170]
 
-We also want to see when the Scheduler returned no transactions, so we will add a logger to the _Default_ condition. From you Mule Palette, drag the logger component to the canvas
+Check to see when the Scheduler returned no transactions.
+
+. Add a logger to the *Default* condition. 
+
+. From you Mule Palette, drag the *logger* component to the canvas.
 
 image:demo_poll_edi_events/image27_demo_poll_edi_events.png[image,width=306,height=254]
 
-Select and review the properties, and enter an appropriate message
+[start=3]
+
+. Select and review the properties, and enter an appropriate message.
 
 image:demo_poll_edi_events/image28_demo_poll_edi_events.png[image,width=601,height=520]
 
-If transactions were retrieved (_When_ condition is true) we need to iterate over all the transactions that have been retrieved. For this we will drag the _For Each_ component from out palette, to our canvas.
+If transactions are retrieved (*When* condition is true), iterate over all transactions that are retrieved.
```

Comment:
It's really unusual to restart numbering without having a heading. But let's discuss.